oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/ElasticIndexDefinition.java

```diff
@@ -114,6 +117,7 @@ public class ElasticIndexDefinition extends IndexDefinition {
     public final int numberOfShards;
     public final int numberOfReplicas;
     public final int[] queryFetchSizes;
+    public final Integer trackTotalHits;
```

Comment:
Shouldn't this be "long" ?

Reply:
No, the `SearchRequestBuilder` requires an `Integer` (see line 224).